semanticdb/javac/src/main/scala/scala/meta/internal/semanticdb/javac/SemanticdbGen.scala

```diff
@@ -0,0 +1,58 @@
+package scala.meta.internal.semanticdb.javac
+
+import java.nio.file.{Files, Path}
+import javax.lang.model.element.TypeElement
+import scala.collection.mutable
+import scala.meta.internal.{semanticdb => s}
+import scala.meta.internal.semanticdb.Scala._
+import scala.meta.internal.semanticdb.javac.semantics._
+import scala.meta.internal.io.PathIO
+
+class SemanticdbGen(relSourcePath: Path, toplevels: Seq[TypeElement]) {
+
+  private val infos = mutable.ListBuffer[s.SymbolInformation]()
+
+  private lazy val relOutputPath =
+    PathIO.toUnix(relSourcePath.toString + ".semanticdb")
+
+  def populate(): Unit = {
+
+    def infoForPackage(pkgSym: String): s.SymbolInformation =
+      s.SymbolInformation(
+        symbol = pkgSym,
+        language = s.Language.JAVA,
+        kind = s.SymbolInformation.Kind.PACKAGE,
+        name = pkgSym.desc.name,
+        accessibility = Some(
```

Comment:
Package should only populate the fields `symbol`, `kind`, `name` 

https://github.com/scalameta/scalameta/blob/master/semanticdb/semanticdb3/semanticdb3.md#root-package

Reply:
Metacp also populates the language and accessibility fields, so I added this and it expanded the diff with metacp.